### ISSUE TITLE
[Suggestion] Lowered roads instead of raised roads? Sidewalks?

### ISSUE THREAD

**Issue by Huntermeign** [StrangeLoopGames/EcoIssues#3582]
Not sure if it was suggested or not because its not exactly an issue.

With the way the roads are now, the stone roads are raised high enough to block entrances to doors, so we make sidewalks with mortared stone.

Problem is the roads look too high cosmetically, and digging the road down an extra block prevents carts from moving off the road.

Also, I was hoping for a sidewalk type block. One that would look more flush with the current roads, still allow players to enter doors, and maybe gives a speed boost while walking

Or you could place down roads with a hammer and have the option of either sidewalk or road.

**Comment by Nes112** [comment]
@Huntermeign This  is a game design decision as to whether you should have to dig up the current road to place a new road block at the same level. Personally, I like the system the way that it is. Yes, it's more trouble, but it is a good opportunity for contract work for shovel specialists. The main one, really.

As for the sidewalks, the problem with that is there is nothing stopping people from making their roads out of sidewalks. Should carts go slower on sidewalks vs. roads? That doesn't make much sense either. I like the idea though.

Then there's the stone road/dirt interface. Road tamper is needed to make that transition smooth for carts.

**Comment by CompactDisc** [comment]
I believe what Hunter is trying to say, is that he would like the road block models to be slightly lower.
So they don't extend above the ground next to it.

You'd still have to dig them out, but right now they take up 1.05 blocks or something, thus blocking any doors next to the road from working. If lowered, that problem would be fixed.
It would come with the added benefit of allowing sidewalks to be built manually by simply placing mortared stone. next to the road, since it will be flush with or above the road.

TLDR; Please lower the road a bit so it's flush with, or slightly lower than, the surrounding blocks.

**Comment by Huntermeign** [comment]
Yes this is for the model itself. I have no problem digging up dirt roads to place down stone or asphalt.

**Comment by MarjieAVolk** [comment]
Upping the priority of this, since as mentioned it is actually blocking people going through doors when the road goes up to the door
